src/test/lib/optimizer/strategy/predicate_reordering_rule_test.cpp

```diff
@@ -269,4 +262,59 @@ TEST_F(PredicateReorderingTest, SimpleValidateReorderingTest) {
   EXPECT_LQP_EQ(actual_lqp, expected_lqp);
 }
 
+TEST_F(PredicateReorderingTest, DoNotReorderMultiPredicateSemiAndAntiJoins) {
+  // The semi-/anti-joins filter the left input relation, the PredicateNode does not. However, do not reorder the nodes
```

Comment:
Cannot follow the comment. There are multiple PredicateNodes, but you talk about one. And why do PredicateNodes not filter the left input?

Are you talking selectivities here?